lib/ansible_tower_client/job_template.rb

```diff
@@ -10,12 +10,19 @@ def initialize(api, raw_body)
       super
     end
 
-    def launch(vars = {})
+    def launch(options = {})
```

Comment:
I would prefer `launch(limit = nil, vars = {})`
If to use options, it should have format `:limit => 'some condition', :extra_vars => {the extra vars}`